README.md

```diff
@@ -30,43 +30,45 @@ Authors 👷:
 Here's how to get the application up and running for development
 
 ### Prerequisites
-* Got Git installed
-* Got Docker installed
+* Got `Git` installed
+* Got `Docker` installed
```

Comment:
I think we should stick to using backticks for commands/code snippets and maybe just use bold if we want to highlight names/words
```suggestion
* Got **Git** installed
* Got **Docker** installed
```

Reply:
Yeah i changed it.